src/main/java/htsjdk/variant/variantcontext/FastGenotype.java

```diff
@@ -29,7 +29,7 @@
 import java.util.Map;
 
 /**
- * This class encompasses all the basic information about a genotype.  It is immutable.
+ * This class encompasses all the basic information about a genotype.  It is unmodifiable, but not immutable.
```

Comment:
`"unmodifiable, but not immutable"` is a rather confusing phrase. I'd replace with something more explicit like `For the sake of performance, it does not make a copy of the Collections/arrays it's constructed from, and so subsequent changes to those Collections/arrays will be reflected in the FastGenotype object`